create-custom-map-style/README.md

```diff
@@ -33,9 +35,11 @@ The following walk through is split into 2 parts:
 
     wget https://github.com/maputnik/editor/releases/download/v1.7.0/maputnik-darwin.zip
     unzip maputnik-darwin.zip
-    chmod 755 maputnik
+    chmod +x maputnik
     ```
 
+    Again this is Mac/Linux specific and while there is a `wget` command for windows, there is no `unzip` or concept of file modes.
```

Comment:
I meant that the Maputnik binary ("darwin") is Mac-specific. WSL may make the other parts work on Windows.

Reply:
Ok I updated the README to explain that, thank you